6-data-storage/02-localstorage/article.md

```diff
@@ -1,81 +1,80 @@
 # LocalStorage, sessionStorage
 
-Web storage objects `localStorage` and `sessionStorage` allow to save key/value pairs in the browser.
+Los objetos de almacenaje web `localStorage` y `sessionStorage` permiten guardar pares de clave/valor en el navegador
 
-What's interesting about them is that the data survives a page refresh (for `sessionStorage`) and even a full browser restart (for `localStorage`). We'll see that very soon.
+Lo que es interesante sobre ellos es que los datos sobreviven a una recarga de página (en el caso de `sessionStorage`) y hasta un reinicio completo de navegador (en el caso de `localStorage`). Lo veremos en breves.
 
-We already have cookies. Why additional objects?
+Ya tenemos cookies. ¿Por qué tener objetos adicionales?
 
-- Unlike cookies, web storage objects are not sent to server with each request. Because of that, we can store much more. Most browsers allow at least 2 megabytes of data (or more) and have settings to configure that.
-- The server can't manipulate storage objects via HTTP headers, everything's done in JavaScript.
-- The storage is bound to the origin (domain/protocol/port triplet). That is, different protocols or subdomains infer different storage objects, they can't access data from each other.
+- Al contrario que las cookies, los objetos de almacenaje web no se envian al servidor en cada petición. Debido a esto, podemos almacenar mucha más información. La mayoría de navegadores permiten almacenar, como mínimo, 2 megabytes de datos (o más) y tienen opciones para configurar éstos límites.
+- El servidor no puede manipular los objetos de almacenaje via cabezeras HTTP, todo se hace via JavaScript.
+- El almacenaje está vinculado al orígen (al triplete dominio/protocolo/puerto). Esto significa que distintos protocolos o subdominios tienen distintos objetos de almacenaje, no pueden acceder a otros datos que no sean los suyos.
 
-Both storage objects provide same methods and properties:
+Ámbos objetos de almacenaje proveen los mismos métodos y propiedades:
 
-- `setItem(key, value)` -- store key/value pair.
-- `getItem(key)` -- get the value by key.
-- `removeItem(key)` -- remove the key with its value.
-- `clear()` -- delete everything.
-- `key(index)` -- get the key on a given position.
-- `length` -- the number of stored items.
+- `setItem(clave, valor)` -- almacena un par clave/valor.
+- `getItem(clave)` -- coje el valor para una clave.
```

Comment:
Great job!! 🎉 
I have a small suggestion, is just related to  this section in particular  
Maybe try doing a better usage of "**Get**" which could be translated as --- (get the value by key)  "**Obtiene el valor por medio de la clave.**"  
EN"**get the key on a given position**" --    ES-"**Obtiene la clave de una posición dada**" (determinada)
 EN-"**get**" -  ES-"**Obtener**" in some countries "**coje**" is not interpreted as it should.

thank you and keep it up!